projects/sbb-esta/angular-public/file-selector/src/_file-selector.scss

```diff
@@ -11,18 +11,30 @@
 
 @mixin fileSelectorButton() {
   @include buttons;
+  @include secondaryButton;
 
-  &:not(.sbb-file-selector-disabled) {
-    @include secondaryButton;
-    @include arrowbuttons;
+  @if $sbbBusiness {
+    .sbb-svgsprite-icon {
+      display: none;
+    }
 
+    &.sbb-file-selector-disabled {
+      background-color: $sbbColorMilk;
+      color: $sbbColorGraphite;
+      &:hover {
+        background-color: $sbbColorMilk;
+        color: $sbbColorGraphite;
+      }
+    }
+  } @else {
+    @include arrowbuttons;
     .sbb-svgsprite-icon {
       @include svgIconColor($buttonSecondaryColor);
     }
-  }
-  // Disabled button
-  &.sbb-file-selector-disabled {
-    @include disabledButton;
+
+    &.sbb-file-selector-disabled {
+      @include disabledButton;
```

Comment:
Only a question, wouldn't it be better to define global disabledButton style for business than defining in file selector style? so one can include it everywhere needed. @kyubisation your opinion?

Reply:
@jeripeierSBB For me, that's the correct solution. Because in design, the button of the file-selector is a normal [secondary button](https://digital.sbb.ch/en/webapps/components/button) with it's different status.